number num1
number num2
---
number sum